wolfssl/internal.h

```diff
@@ -1449,7 +1449,11 @@ enum Misc {
     COMP_LEN     =  1,         /* compression length      */
     CURVE_LEN    =  2,         /* ecc named curve length  */
     KE_GROUP_LEN =  2,         /* key exchange group length */
-    SERVER_ID_LEN = 20,        /* server session id length  */
+#if defined(NO_SHA) && defined(NO_SHA256)
```

Comment:
Also need !defined on the NO_SHA256 here I think

Reply:
Fixed.
Thanks!